new-client/src/plugins/Fir/FirImport.js

```diff
@@ -52,7 +54,6 @@ export default class FirImport {
   };
 
   translateImportedFeature(feature) {
-    // got this from the Draw plugin...
     var coordinates = feature.getGeometry().getCoordinates(),
```

Comment:
`const coordinates = ...`?

Reply:
Indeed, it's a copyNpaste bonus. I'll fix!